src/Options/index.js

```diff
@@ -292,6 +295,15 @@ export interface IdempotencyOptions {
   ttl: ?number;
 }
 
+export interface SecurityChecksOptions {
+  /* If true if Parse Server should self-check the security of its current configuration.
+  :DEFAULT: false */
+  enableSecurityChecks: ?boolean;
+  /* Is true if security warnings should be written to logs. This should only be enabled temporarily to not expose weak security settings in logs.
+  :DEFAULT: false */
+  enableLogOutput: ?boolean;
+}
```

Comment:
Here may be we can enable by default the option in NODE ENV development since it's helpful for developer to show him some security warnings as soon as possible.

In NODE env Prod let's disable this by default and let the developer the choice to activate it.

Reply:
I'm not sure about changing default values depending on environment. This could create more confusion for the developer as to how to set up a production environment coming from a development environment. I also don't think we currently do that with any other option?

Reply:
yes let's remove resolve this one !